doc/releases/release-notes-3.4.rst

```diff
@@ -486,6 +486,8 @@ Libraries / Subsystems
   * The FAT FS initialization order has been updated to match LittleFS, fixing an issue where
     attempting to mount the disk in a global function caused FAT FS to fail due to not being registered beforehand.
     FAT FS is now initialized in POST_KERNEL.
+  * Added :kconfig:option:`CONFIG_FS_LITTLEFS_FMP_DEV` to enable possibility of using
+    LittleFS for block devices only, e.g. without Flash support.
```

Comment:
Note that the option is default y too keep previous behaviour.

Reply:
thanks! Added